### PR TITLE
update forestry settings

### PR DESCRIPTION
**Pull request by flohump**
## :bird: Description of this PR :bird:

- **default.cfg and scenario_config.csv** Default for cfg$gms$c56_emis_policy changed changed from `redd+natveg_nosoil` to `reddnatveg_nosoil`,i.e. timber plantations are excluded from carbon pricing by default
- **default.cfg** Default for cfg$gms$c56_cprice_aff from `forestry_vegc` to `secdforest_vegc`
- **scenario_config.csv** settings for cfg$gms$s35_secdf_distribution in `ForestryEndo` and `ForestryExo` changed from `2` to `0`
- **scenario_config.csv** removed erroneous setting `cc` from column `input`

## :wrench: Checklist for PR creator :wrench:

- [x] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts
  - **Medium risk**: Uncritical changes in the model core (e.g. moderate modifications in non-default realizations)
  - **High risk**: Critical changes in model core or default settings (e.g. changing a model default or adjusting a core mechanic in the model)

- [x] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [x] Document changes 
  - Add changes to `CHANGELOG.md`
  - Where relevant, put In-code documentation comments
  - Properly address updates in interfaces in the module documentations
  - run [`goxygen::goxygen()`](https://github.com/pik-piam/goxygen) and verify the modified code is properly documented

- [x] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
  - **High risk**:
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings
    - Default run from the PR target branch for comparison
    - Provide relevant comparison plots (land-use, emissions, food prices, land-use intensity,...)

### :chart_with_downwards_trend: Performance changes :chart_with_upwards_trend:
  
  - Current develop branch default : ** mins
  - This PR's default :  ** mins

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [x] RSE review done (at least 1)

### CODE REVIEW

**Review by tscheypidi — APPROVED** [review]

**Comment by merfort** [comment]
Thanks, Flo! Just one question: In `config/scenario_config.csv`, was it intended to also remove the `cc` in the `input` column from `c59_som_scenario`? I don't see, how that is related to the forestry changes.

**Comment by flohump** [comment]
> Thanks, Flo! Just one question: In `config/scenario_config.csv`, was it intended to also remove the `cc` in the `input` column from `c59_som_scenario`? I don't see, how that is related to the forestry changes.

`cc` was removed from the column `input` by intention.
I added a comment in the CHANELOG file.

File: CHANGELOG.md
Comment (merfort) [inline CHANGELOG.md:1]: Perfect, thanks a lot!

**Review by merfort — APPROVED** [review]
Looks all good to me.

**Pull request merged**